src/pages/LoginAndRegister/LoginAndRegister.vue

```diff
@@ -23,12 +26,26 @@ export default {
 	}
 };
 </script>
+
 <style lang="scss" scoped>
 @import 'settings';
 
+.page-content {
+	padding: 1.625rem rem-calc(10);
+}
+@include breakpoint(medium) {
+	padding: 1.625rem 0;
+}
+
+// Control form layouts from parent to allow independence in other layouts
+.login-form,
+.register-form {
+	max-width: 280px;
```

Comment:
Should probably make this max-width: rem-calc(280);

Reply:
can do